tests/sftp_pdr_parse_ingest.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+const path = require('path');
+const test = require('ava');
+const fs = require('fs-extra');
+const { fetchMessageAdapter } = require('../packages/deployment/lib/adapter');
+const {
+  runWorkflow,
+  copyCMAToTasks,
+  deleteCMAFromTasks,
+  messageBuilder
+} = require('../packages/integration-tests/local');
+const { randomString, createQueue } = require('../packages/common/test-utils');
+const {
+  recursivelyDeleteS3Bucket,
+  s3,
+  sqs,
+  receiveSQSMessages
+} = require('../packages/common/aws');
+const workflowSet = require('./fixtures/workflows/pdr_parse_ingest.json');
+const collections = require('./fixtures/collections.json');
+const providers = require('./fixtures/providers.json');
+
+// unfortunately t.context is not available in test.before
+// this is fixed in ava 1.0.0 but it has a lot of breaking
+// changes. The global variables below help with passing messages
+// around between before and after hooks.
+const context = {};
+const cmaFolder = 'cumulus-message-adapter';
+
+
+test.before(async() => {
+  context.internal = randomString();
+  context.stack = randomString();
+  context.templates = {};
+  await s3().createBucket({ Bucket: context.internal }).promise();
+
+  // download and unzip the message adapter
+  const gitPath = 'cumulus-nasa/cumulus-message-adapter';
+  const filename = 'cumulus-message-adapter.zip';
+  context.src = path.join(process.cwd(), 'tests', `${randomString()}.zip`);
+  context.dest = path.join(process.cwd(), 'tests', randomString());
+  await fetchMessageAdapter(null, gitPath, filename, context.src, context.dest);
```

Comment:
Since these lines of code are duplicated in `tests/ftp_pdr_parse_ingest.js` could we extract it into a function in`testUtils` (or the `fetchMessageAdapter` function itself)?